### DISCUSSION TITLE
View loaded or created dxf file?

### DISCUSSION THREAD

**Discussion by Fikri-coder** [haplokuon/netDxf#436] [Q&A]
Hello,
just to say this is wonderfull project. Thank you for that.
I know how to read and create dxf now.
Is there any example or idea how to show dxf file on the form, center it and zoom, to create viewport?

Thank you

**Comment by haplokuon** [comment]
This kind of questions fall outside the scope of this library. It is more dependent on the graphics engine you use and, probably, you will have a better answer there.
